L2 first draft

ambasador77 opened the pull request:
please review @ex-man, @MatejBreja  (compare with GeneralInsurance_Class/Lessons/Lesson2/README_old.md)

- is there anything missing that should be there?
- should there be also direct walk-through for the data prep exercise?
- should there be any text for the 3rd section? (look for underperformers)

ambasador77 committed:
Merge pull request #5 from MatejBreja/master

update from Matej

ambasador77 committed:
keeping backup of an old readme file

ambasador77 committed:
creating a new read me with links to the files

ambasador77 committed:
L2: intro to losses and expenses

ambasador77 committed:
Update L2_losses_and_expenses.md

ambasador77 committed:
Update L2_loss_and_expense_ratios.md

ex-man commented on Lessons/Lesson2/Support/L2_losses_and_expenses.md:
```diff
@@ -1 +1,25 @@
+
+The concept of insurance can be understood as a method of hedging financial losses of a group of insured that are exposed to the same kind of loss. The sharing of the losses is covered by a common fund that consists of a sum of financial contributions from the group of insured.
+An insurer is a company that sells insurance policies that work as an entry ticket to the risk pooling and the fee paid is called premium. 
+For example it is almost certain that within a single year in a certain large urban area there will be a car accident. Also it is almost certain, that not every car in the area will have an accident. Now if a car owner happens to be the one to have an accident, he will have to cover the losses he suffers. However, if the owner agrees to share his risk of suffering loss with a group of similar car ...
```
Own owners... :)

ex-man commented:
I like it. A bit too long but good reading... One word duplication to be removed...

ambasador77 committed:
L2 - bold and paragraphs update

key words are highlighted by bold, text is now split to several smaller paragraphs

ambasador77 committed:
L2 underwriting result and combined ratio

updated to contain the definition of underwriting result and combined ratio

ambasador77 committed:
exercise update in cooperation with @cvoky

ambasador77 commented:
together w/  @cvoky we've updated the L2 exercies, please review @MatejBreja, @ex-man
